### ISSUE TITLE
Unblocking an IP crashes the whole site

### ISSUE THREAD

**Issue by drbobw46** [codeling/bfstop#194]
I have been using BFStop for many years, currently version 1.4.4 on J4.3.1. Recently I have found that unblocking a blocked IP sends me to a red warning screen 'Oops something went wrong'. My workaround is to edit the blocked entry and set the unblock to occur after 5 minutes.
What other information or access do you require?
Bob

**Comment by codeling** [comment]
Most likely a duplicate of https://github.com/codeling/bfstop/issues/191.
I did not manage to provide the release promised there yet - but this will happen soon!

**Comment by codeling** [comment]
Finally got a release out that fixes this issue [1.4.6](https://github.com/codeling/bfstop/releases/tag/1.4.6). This release should also get distributed already through Joomla's update system.
Let me know if you still encounter any issues with that version!

**Comment by drbobw46** [comment]
Many thanks. I'll install and test.
